source/backend/dal/Repositories/ManagementFileRepository.cs

```diff
@@ -315,6 +316,67 @@ public List<PimsManagementFileTeam> GetTeamMembers()
                 .ToList();
         }
 
+        public List<PimsManagementFileContact> GetContacts(long managementFileId)
+        {
+            User.ThrowIfNotAllAuthorized(Permissions.ManagementView);
```

Comment:
I believe we do permission checking at the service level only and do not duplicate the check in the repos - please remove